testdata/helm/memcached-operator/.gitignore

```diff
@@ -7,6 +7,16 @@
 *.dylib
 bin
 
+# Test binary, build with `go test -c`
```

Comment:
@camilamacedo86 : These changes in samples are not a part of this PR which involves updating Go version in SDK. Shall I create a new PR for these? The samples in master need to be updated to use 1.0.1 instead of 1.0.0